### DISCUSSION TITLE
Mouse cursor doesn't move under Fedora Linux when using Wayland

### DISCUSSION THREAD

**Discussion by nxtroox** [ppy/osu#23950] [Q&A]
### Type

Game behaviour

### Bug description

The cursor doesn't move when I move my mouse. I can still do left and right clicks. It worked on X11 but not on Wayland.

### Version

2023.617.0

### Logs

[input.log](https://github.com/ppy/osu/files/11778861/input.log)
[network.log](https://github.com/ppy/osu/files/11778862/network.log)
[performance.log](https://github.com/ppy/osu/files/11778863/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11778864/runtime.log)
[database.log](https://github.com/ppy/osu/files/11778865/database.log)

**Comment by ludverse** [comment]
can confirm and its annoying the crap otta me

**Comment by jieran233** [comment]
Try adding this to `/etc/environment` then reboot?
`SDL_VIDEODRIVER=wayland`
I'm not sure whether it can take effect, but I hope it can help you. If not, reverse the change please.
